src/include/stir/recon_buildblock/DataSymmetriesForBins_PET_CartesianGrid.inl

```diff
@@ -479,7 +479,7 @@ find_basic_view_segment_numbers(ViewSegmentNumbers& v_s) const
 
 bool  
 DataSymmetriesForBins_PET_CartesianGrid::
-find_basic_bin(int &segment_num, int &view_num, int &axial_pos_num, int &tangential_pos_num) const 
+find_basic_bin(int &segment_num, int &view_num, int &axial_pos_num, int &tangential_pos_num, int &timing_pos_num) const
```

Comment:
Pass `bin`?

Reply:
A change to this functionality is out of scope of this PR. `inline bool find_basic_bin(Bin& b) const;` does exist and is a private member but its functionality is fairly specific.

Reply:
sure